pandas/core/indexes/base.py

```diff
@@ -1868,7 +1868,7 @@ def _isnan(self):
     @cache_readonly
     def _nan_idxs(self):
         if self._can_hold_na:
-            w, = self._isnan.nonzero()
+            (w,) = self._isnan.nonzero()
```

Comment:
can we add a `[0]` at the end and get rid of the comma instead?

Reply:
Sure done